File: Selenium/Activities/Activity4.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/target-practice")
    print("Title is :"+driver.title)
    # find Third Header on the page
    thirdheader=driver.find_element(By.XPATH,"//h3[@id='third-header']").text
    print("Third header text is :"+thirdheader)
    fifthheader=driver.find_element(By.XPATH,"//h5[@class='ui green header']").value_of_css_property("color")
    print("Third header text is :"+fifthheader)
    classes=driver.find_element(By.CLASS_NAME,"violet").get_attribute("class")
    print("Violet button classes are :"+classes)
    greyBtnText=driver.find_element(By.CLASS_NAME,"grey").text
    print("Grey btn text is :"+greyBtnText)

File: Selenium/Activities/Activity4.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity4 {
    public static void main(String[] args) {
        WebDriver driver=new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/target-practice");
        System.out.println("title is :"+driver.getTitle());
        //find Third Header on the page
        String thirdHeaderText=driver.findElement(By.xpath("//h3[@id='third-header']")).getText();
        System.out.println("Third header name is :"+thirdHeaderText);
        String fifthHeaderColor=driver.findElement(By.xpath("//h5[@class='ui green header']")).getCssValue("color");
        System.out.println("Fifth header color is :"+fifthHeaderColor);
        String className=driver.findElement(By.className("violet")).getAttribute("class");
        System.out.println("Violet button class name is :"+className);
        String greyBtnText=driver.findElement(By.className("grey")).getText();
        System.out.println("Grey button text is :"+greyBtnText);
        driver.close();

    }
}
